clients/cli/proto

```diff
@@ -0,0 +1 @@
+../../proto
```

Comment:
I don't understand this

Reply:
There were two proto files checked in, now there is one and a soft link

Reply:
Removed the soft link since I think it won't work on WSL. I'm using relative paths now.